### ISSUE TITLE
Hyperparameter-Tuning

### ISSUE THREAD

**Issue by simon376** [simon376/ds-recommender-project#2]
decide on one or two models and try to hypertune the parameters, using different: 
* loss functions
* Embedding sizes
* learning rates (adaptable? LR scheduler?)
* 
* (Datasets: e.g. remove string data and see if it helps/hurts)
